ci/code_checks.sh

```diff
@@ -141,7 +141,7 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
     MSG='Check for python2 new-style classes' ; echo $MSG
-    invgrep -R --include="*.py" -E "class\s\S*\(object\):" pandas scripts
+    invgrep -R --include="*.py" --include="*.pyx" -E "class\s\S*\(object\):" pandas scripts
```

Comment:
Would make sense to just have one include flag like in the other PR

Reply:
This follows the conventions of the code_checks.sh file. I also find it more readable than the other convention...